package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "hatchways",
+  "version": "1.0.0",
+  "description": "This starter repo will be used for building applications using React, Material-UI, React-Router, Node, & Express.js.",
+  "main": "index.js",
+  "dependencies": {
+    "react-scripts": "^4.0.2"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "start": "node index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
```

Comment:
you can remove this and package-lock.json. There's no package.json needed in the root folder